save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/utils/DateTimeUtils.kt

```diff
@@ -40,6 +40,15 @@ fun LocalDateTime.toUnixCalendarFormat(timeZone: TimeZone = TimeZone.UTC) = toIn
             .minute.plusZero()}"
     }
 
+/**
+ * @param timeZone timezone to print the date time in
+ * @return string representation of [LocalDateTime] in [Thursday, 1 Jan 1970 00:00:00] format
+ */
+fun LocalDateTime.toUnixCalendarFormatWithSeconds(timeZone: TimeZone = TimeZone.UTC) = toInstant(TimeZone.UTC).toLocalDateTime(timeZone)
```

Comment:
in future need to merge it with the second method

Reply:
with flag

Reply:
Fixed